source/common_modules.f90

```diff
@@ -197,12 +197,16 @@ integer,          save :: curveff    = 0    ! Enable the curvature effect in a c
   integer,          save :: iicav      = 0    ! Used between runcav and runda
   integer,          save :: ition      = 0    ! Transition energy switch:
   integer,          save :: idp        = 0    ! Synchrotron motion
-  integer,          save :: ncy        = 0    ! Number of cavity locations
   integer,          save :: ixcav      = 0    ! Stores ix, presumably for cavity
   integer,          save :: icode      = 0
   integer,          save :: idam       = 0
   integer,          save :: its6d      = 0
 
+  ! RF Cavities
+  integer,          save :: icy        = 0    ! Accelerating cavity: Number of "CAV" locations in STRUCT
+  integer,          save :: ncy        = 0    ! Accelerating cavity: Number of cavity locations in SING
```

Comment:
Cavity location in SING? Should this be in STRUCT?

Reply:
The value is incremented in SING parsing.

Reply:
`ncy` is `icy*mper`. Updated comment.